can_tools/scrapers/official/base.py

```diff
@@ -1212,3 +1213,52 @@ def get_dataset(self, body, url) -> str:
         """Accepts JSON body, and a url to post to the data studio batched URL"""
         rawJson = str(requests.post(url, json=body).content)
         return rawJson
+
+
+class ETagCacheMixin:
+    """Mixin class to add the ability to check whether a dataset has been updated since last viewed.
+
+    This is used in the `create_cached_flow_for_scraper` flow, to determine whether
+    to ingest the data (execute the fetch, normalize, and put methods).
+    """
+
+    cache_url: str
+    cache_name: str
+    cache_file: Path = Path(__file__).parents[1] / "cache_scraper_versions.json"
+
+    @property
+    def etag(self):
+        res = requests.get(self.cache_url)
```

Comment:
Yeah this would've been ideal. Unfortunately, I tried this for the `CDCCountyVaccine2` class and I got the error message:
```
{... 'X-Error-Code': 'invalid_request', 'X-Error-Message': 'HEAD is not supported' ...}
```
using GET is slower, but it should work for all the URLs that we need

Reply:
Ah, bummer. Sounds good though.